typescript/src/client/cdp.ts

```diff
@@ -107,6 +107,8 @@ For more information, see: https://github.com/coinbase/cdp-sdk/blob/main/typescr
       sourceVersion: version,
     });
 
+    import("../analytics.js").then(module => module.AnalyticsConfig.set(apiKeyId));
```

Comment:
Importing like this to avoid a circular dependency. This works as expected

Reply:
Nit: Could we add a comment explaining how this avoids the circular dependency

Reply:
Ok I actually removed this dynamic import in favor of a regular import – Node's caching and partial export features allow the circular dependency to work, only loading `analytics.ts` is [the very first thing we do](https://github.com/coinbase/cdp-sdk/blob/main/typescript/src/index.ts#L1). It is a bit weird though, in particular the fact that outwardly it looks like we run analytics code even when opt out flag is set, so I think I want to change that.